webapp/package-lock.json

```diff
@@ -538,18 +538,18 @@
     },
     "are-we-there-yet": {
       "version": "1.1.5",
-      "resolved": "https://registry.npmjs.org/are-we-there-yet/-/are-we-there-yet-1.1.5.tgz",
-      "integrity": "sha512-5hYdAkZlcG8tOLujVDTgCT+uPX0VnpAH28gWsLfzpXYm7wP6mp5Q/gYyR7YQ0cKVJcXJnl3j2kpBan13PtQf6w==",
+      "resolved": "https://box.jfrog.io/box/api/npm/boxnpm/are-we-there-yet/-/are-we-there-yet-1.1.5.tgz",
```

Comment:
jfrog -> registry again. Same problem I had before!

Reply:
Great catch! Let me change my npm config and re-run.